Pages/Hotels/hotelsHome.js

```diff
@@ -0,0 +1,481 @@
+    childIncrease = "//body/div[@id='app-blossom-flex-ui']/div[@id='app-layer-manager']/div[@id='app-layer-base']/div[1]/div[2]/div[1]/div[1]/section[1]/form[1]/div[1]/div[1]/div[3]/div[1]/div[2]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[1]/button[2]"
+    childDecrease = "//*[@id='traveler_selector_children_step_input-0-decrease-title']"
+    
+    removeChildrenButton = "//input[contains(@aria-label,'Children')]/preceding-sibling::button";
+    addChildrenButton = "//input[contains(@aria-label,'Children')]/following-sibling::button"
+    addAdultsButton = "//input[@aria-label='Adults ']/following-sibling::button/span"
+
+    child1Age = '//select[@id="age-traveler_selector_children_age_selector-0-0"]'
+    child2Age = '//select[@id="age-traveler_selector_children_age_selector-0-1"]'
+    child3Age = '//select[@id="age-traveler_selector_children_age_selector-0-2"]'
+    child4Age = '//select[@id="age-traveler_selector_children_age_selector-0-3"]'
+    child5Age = '//select[@id="age-traveler_selector_children_age_selector-0-4"]'
+    child6Age = '//select[@id="age-traveler_selector_children_age_selector-0-5"]'
+    
+    // child1AgeSelect = '//select[@id="age-traveler_selector_children_age_selector-0-0"]//option[@value="0"]'
+    // child2AgeSelect = '//select[@id="age-traveler_selector_children_age_selector-0-1"]//option[@value="4"]'
+    // child3AgeSelect = '//select[@id="age-traveler_selector_children_age_selector-0-2"]//option[@value="7"]'
+    
+    totalAdults = '//input[@id="traveler_selector_adult_step_input-0"]'
+    totalChildren = '//input[@id="traveler_selector_children_step_input-0"]'
+    numOfChildren = "//input[contains(@aria-label,'Children')]"
+
+    travelersDoneButton = "//button[text()='Done']"
+
+    numOfTravelers = "//button[contains(text(),'travelers')]"
+    numOfDropdownsForChildren = "//select[contains(@name,'child-traveler_selector')]"
+
+    phoneNumberError = '//*[@id="phoneNumber-error"]'
+    getTheAppBtn = '//*[@id="submitBtn"]'
+    phoneNumberLocator = '//*[@id="phoneNumber"]'
+    
+    languageOptions = '//select[@id="language-selector"]'
+
+    englishButton ='//div[contains(text(),"English")]'
+    spanishOption = '//option[contains(text(),"Español (Estados Unidos)")]'
+    saveButton ='//button[contains(text(),"Save")]'
+    espanolButton = '//div[contains(text(),"Español")]'
+    englishOption =  '//option[contains(text(),"English (United States)")]'
+    spanishSaveButton = "//button[contains(text(),'Guardar')]"
+ 
+
+    property = '//div[contains(text(),"List your property")]'
+    wouldYouLike = '//p[contains(text(),"What would you like to list?")]'
+    lodging = '//p[contains(text(),"Lodging")]'
+    privateResidence = '//p[contains(text(),"Private residence")]'
+    
+    step1 = '//div[contains(text(),"Step 1 of 3")]'
+    step2 = '//div[contains(text(),"Step 2 of 3")]'
+    step3 = '//div[contains(text(),"Step 3 of 3")]'
```

Comment:
These selectors names are not descriptive. It could be something like step1of3Text.